install.bat

```diff
@@ -0,0 +1,78 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0. Make sure you have it installed."
+
+SET /P check_repo="Do you want to clone the repository [y/n] : "
+IF %check_repo%==y ( ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git 
+) ELSE ( 
+ECHO "Make sure that you have the repository and the install.bat file in the same folder!" 
+)
+
+cd mentorship-backend
```

Comment:
Hey @Hardik0403, this should not be executed if we're not cloning the repo. You would want to move it inside the if-else clause above!

Reply:
I assumed that install.bat file is outside the mentorship-backend folder with the user, I have now fixed this.